mlir/python/CMakeLists.txt

```diff
@@ -403,6 +403,17 @@ declare_mlir_dialect_python_bindings(
   GEN_ENUM_BINDINGS_TD_FILE
     "dialects/VectorAttributes.td")
 
+declare_mlir_dialect_python_bindings(
+  ADD_TO_PARENT MLIRPythonSources.Dialects
+  ROOT_DIR "${CMAKE_CURRENT_SOURCE_DIR}/mlir"
+  TD_FILE dialects/FIROps.td
+  SOURCES dialects/fir.py
+  DIALECT_NAME fir
+  GEN_ENUM_BINDINGS_TD_FILE
+    "dialects/FIRAttrs.td"
+  EXTRA_INCLUDES "${CMAKE_CURRENT_SOURCE_DIR}/mlir/../../../flang/include"
+)
+
```

Comment:
Flang is a separate project from MLIR, I don't see this belonging here right now.

Reply:
Of course - like I mentioned in the description - this is just a test of the extra argument to the cmake function and I will drop the commit after CI passes.